src/arkode/arkode.c

```diff
@@ -272,6 +272,14 @@ int ARKodeSStolerances(void* arkode_mem, sunrealtype reltol, sunrealtype abstol)
     return (ARK_ILL_INPUT);
   }
 
+  /* Ensure that vector supports N_VAddConst */
+  if (!ark_mem->tempv1->ops->nvaddconst)
```

Comment:
This also needs to be checked in `arkCheckNvector2` in case the user is running with the default tolerances (in which case this function is not called)

Reply:
This same check is needed in `ARKodeResStolerance` and a similar check in `arkCheckNvector2`